### PR TITLE
16. youtube video section impl

### PR DESCRIPTION
**Pull request by petroborovets**
Screenshot 2024-05-12 at 21 50 13

### CODE REVIEW

File: src/components/common/post/section/child/list/ListPostSection.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import s from "./ListPostSection.module.css"
+import PropTypes from "prop-types";
+
+function ListPostSection({list}) {
+    return (
+        <ul>
+            {
+                list?.map(li =>
+                    <li className={s.post_section_list_child} key={li} dangerouslySetInnerHTML={{__html: li}}/>
```
Comment (boandriy) [inline src/components/common/post/section/child/list/ListPostSection.jsx:9]: key should always be unique, as long as the li is unique - this is ok.

Comment (petroborovets) [reply src/components/common/post/section/child/list/ListPostSection.jsx:9]: And what if it is not unique? 
Reason I am asking is that I really do not want to declare a lot of ids, in those json files with content I created.

**Review by boandriy — APPROVED** [review]

File: src/constants/posts.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
       "author": "Petro Borovets",
       "hoursOnCalls": "5",
       "headSection": {
-        "headingText": "Отже, тут буде коротка історія того як сайт будується або будувався. Так як молодший брат FE дев, а старший брат BE дев, початкова задумка була така, що, молодший практикується у BE частині, а старший брат у FE частині, таким чином перебираючи досвід один в одного, а як складеться чи склалось в результаті - напишу потім :). Мені завжди було цікаво завести сайтик на якому можна буде документувати щось корисне для себе у вигляді фото і тексту, щоб потім можна було із ностальгією перечитувати і згадувати, як то було, такий собі щоденник.",
+        "headingText": "Отже, тут буде коротка історія того як сайт будується або будувався. Так як молодший брат FE дев, а старший - BE дев, початкова задумка була така, що, молодший практикується у BE частині, а старший брат у FE частині, таким чином перебираючи досвід один в одного, а як складеться в результаті - напишу потім :). Мені завжди було цікаво завести сайтик на якому можна буде документувати щось корисне для себе і фіксувати враження подій, щоб потім можна було із ностальгією згадувати старі історії.",
```
Comment (boandriy) [inline src/constants/posts.json:12]: молодший брат вже більше ніж 2 роки працює з BE :)

**Review by boandriy — APPROVED** [review]

File: src/constants/posts.json (hunk 2 above)
Comment (petroborovets) [reply src/constants/posts.json:12]: Who cares.

**Pull request merged**